bpf/lib/conntrack.h

```diff
@@ -406,10 +408,13 @@ static __always_inline int ct_lookup6(const void *map,
 			if (ctx_load_bytes(ctx, l4_off + 12, &tcp_flags, 2) < 0)
 				return DROP_CT_INVALID_HDR;
 
-			if (unlikely(tcp_flags.value & (TCP_FLAG_RST|TCP_FLAG_FIN)))
+			if (unlikely(tcp_flags.value & TCP_FLAG_RST)) {
+				action = ACTION_RESET;
+			} else if (unlikely(tcp_flags.value & TCP_FLAG_FIN)) {
 				action = ACTION_CLOSE;
-			else
+			} else {
 				action = ACTION_CREATE;
+			}
```

Comment:
As the script pointed out, we use kernel style in the BPF datapath code, so no braces unless necessary. For consistency, please switch over to the approach suggested byt he checkpatch script.

Reply:
done